def give(n):
    n+10
    raise ValueError

print(give(10))